lib/server.js

```diff
@@ -108,13 +108,14 @@ export default class Server {
     const listen = [];
     const params = url.searchParams;
 
-    if (url.port) {
+    if (url.port !== '') {
```

Comment:
You're calling the `port` getter twice now, which is not the most efficient solution.